src/Contrib/Zipkin/SpanExporterFactory.php

```diff
@@ -20,4 +20,13 @@ public function create(): SpanExporterInterface
 
         return new Exporter($transport);
     }
+
+    public function type(): string
+    {
+        return 'zipkin';
+    }
+    public function priority(): int
```

Comment:
```suggestion
    }

    public function priority(): int
```

Sorry :grimacing: 
Hopefully there's a lint setting somewhere so I don't look so pedantic :sweat_smile:

Reply:
there is a fixer that can do this:
`'class_attributes_separation' => ['elements' => ['method' => 'one'],]`
It changes a lot of files, so I won't do it in this PR, but did fix the blanks b/w `priority()`

Reply:
this fixer has since been applied to the codebase.